_studio/shared/umc/codec/vc1_dec/src/umc_vc1_dec_task_store.cpp

```diff
@@ -177,16 +177,16 @@ namespace UMC
 
     uint32_t VC1TaskStore::CalculateHeapSize()
     {
-        uint32_t Size = mfx::align_value<uint32_t>(sizeof(VC1FrameDescriptor*)*(m_iNumFramesProcessing));
+        uint32_t Size = mfx::align2_value<uint32_t>(sizeof(VC1FrameDescriptor*)*(m_iNumFramesProcessing));
 
         for (uint32_t counter = 0; counter < m_iNumFramesProcessing; counter++)
         {
                 if (pMainVC1Decoder->m_va)
                 {
-                    Size += mfx::align_value<uint32_t>(sizeof(VC1FrameDescriptorVA_Linux<VC1PackerLVA>));
+                    Size += mfx::align2_value<uint32_t>(sizeof(VC1FrameDescriptorVA_Linux<VC1PackerLVA>));
                 }
                 else
-                    Size += mfx::align_value<uint32_t>(sizeof(VC1FrameDescriptor));
+                    Size += mfx::align2_value<uint32_t>(sizeof(VC1FrameDescriptor));
```

Comment:
Do we still need explicit `uint32_t` here and above?

Reply:
no